File: lang-java/src/test/resources/fixtures/hidden-nested-loop/positive/loop-calls-method-with-loop.java
import java.util.List;

public class OrderService {
    public void processOrders(List<Order> orders) {
        for (Order order : orders) {
            validateItems(order);
        }
    }

    private void validateItems(Order order) {
        for (Item item : order.getItems()) {
            checkInventory(item);
        }
    }

    private void checkInventory(Item item) {
        // point lookup, no loop
    }
}

File: lang-kotlin/src/test/resources/fixtures/hidden-nested-loop/positive/loop-calls-method-with-loop.kt
class OrderService {
    fun processOrders(orders: List<Order>) {
        for (order in orders) {
            validateItems(order)
        }
    }

    private fun validateItems(order: Order) {
        for (item in order.items) {
            checkStock(item)
        }
    }

    private fun checkStock(item: Item) {
        // point lookup
    }
}
